ui/src/app/pages/profile/data-access-panel.tsx

```diff
@@ -1,82 +1,67 @@
-  return <div style={{
-    marginBottom: '0.9rem',
-    display: 'grid',
-    columnGap: '0.25rem',
-    width: 459,
-    gridTemplateColumns: 'fit-content(2rem) fit-content(10rem) 1fr',
-    gridTemplateAreas: `"ctBadge ctLabel ctAvailable"
-                          ". ctPrimary ctPrimary"
-                          ". ctSecondary ctSecondary"`
-  }}>
-    <ControlledTierBadge style={{gridArea: 'ctBadge'}}/>
-    <div style={{...styles.inputLabel, gridArea: 'ctLabel'}}>{AccessTierDisplayNames.Controlled}</div>
-    {Utils.cond<React.ReactElement>(
-      // TODO: Remove update href and remove _blank target from Anchor tags
-      [hasInstitutionalAgreement && userRevoked, () => <React.Fragment>
-          <div style={{ ...styles.dataAccessText, gridArea: 'ctPrimary'}}>Access to controlled tier data is revoked.</div>
-          <div style={{ ...styles.dataAccessText, gridArea: 'ctSecondary'}}>
-            To gain access <StyledAnchorTag style={{textDecoration: 'underline'}}
-              href='about:blank' target='_blank'>contact admin.</StyledAnchorTag>
-          </div>
-        </React.Fragment>],
-      [hasInstitutionalAgreement && isInControlledTier,
-        () => <CheckCircle style={{gridArea: 'ctAvailable'}}color={colors.success} size={23}/>],
-      [hasInstitutionalAgreement, () => <React.Fragment>
-          <div style={{ ...styles.dataAccessText, gridArea: 'ctPrimary'}}>You must complete the Controlled Tier Data Training.</div>
-          <StyledAnchorTag style={{gridArea: 'ctSecondary', textDecoration: 'underline'}}
-            href='about:blank' target='_blank'>Get Started</StyledAnchorTag>
-        </React.Fragment>],
-      () => <React.Fragment>
-          <div style={{gridArea: 'ctPrimary', color: colors.primary}}>{needsAgreementText}</div>
-          <StyledAnchorTag style={{gridArea: 'ctSecondary', textDecoration: 'underline'}}
-            href='about:blank' target='_blank'>Learn More</StyledAnchorTag>
-        </React.Fragment>
-    )}
-  </div>;
-};
+export interface DataAccessPanelProps {
+  userAccessTiers: string[];
+}
+export const DataAccessPanel = (props: DataAccessPanelProps) => {
+  const {userAccessTiers} = props;
+
+  const [navigate, ] = useNavigation();
+
+  const orderedTiers = [
+    AccessTierShortNames.Registered,
+    AccessTierShortNames.Controlled
+  ];
 
-export const DataAccessPanel = ({
-    tiers = [], hasInstitutionalAgreement = false, userRevoked = false
-  }) => {
-  const sectionId = Utils.useId();
-  return <section aria-labelledby={sectionId}>
-    <div id={sectionId} style={styles.title}>Data access</div>
+  const sectionId = useId();
+  return <section aria-labelledby={sectionId} style={{marginLeft: '1rem'}}>
+    <FlexRow id={sectionId}>
+      <div style={styles.title}>Data access</div>
+      <Link style={{marginLeft: 'auto'}} onClick={() => navigate(['data-access-requirements'])}>Manage data access</Link>
```

Comment:
Please use the react-router [Link](https://reactrouter.com/web/api/Link) for navigation instead.

Reply:
ok, thanks.

Do we always want this where `navigate` is currently used?

Reply:
Yes, we do. There's a low priority [ticket](https://precisionmedicineinitiative.atlassian.net/browse/RW-7168) to replace them all.

Reply:
updated here